src/kaleidoscope/driver/mcu/NoMCU.h

```diff
@@ -0,0 +1,30 @@
+/* -*- mode: c++ -*-
+ * driver::MCU::NoMCU -- Dummy MCU driver for Kaleidoscope
```

Comment:
...when would we ever want to use this?

Reply:
Like with `NoOpKeyscanner`: never. It's here to let the base `DeviceDescription` be able to use `NoMCU` instead of `BaseMCU`.